config/kafka-capacity-config.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 # see https://docs.google.com/spreadsheets/d/1W4H1IgDXKv24Sf7aWeqeAqxjKVj8EJB9giYN9obqPgU/edit#gid=493008265
 ingressEgressThroughputPerSec: "2Mi"
```

Comment:
should this change to two separate values?

Reply:
I think we said to leave the config files for now. But on the other hand, that will prevent the fleet manager from running locally right? So maybe better to update it in this PR.